test/e2e/wdio.conf.js

```diff
@@ -2,10 +2,16 @@
 const path = require('path');
 require('@okta/env').config();
 
+// You can use Firefox for e2e testing locally if there is an issue with `chromedriver`
+// You need to have `geckodriver` process running (https://github.com/mozilla/geckodriver/releases)
+const USE_FIREFOX = !!process.env.USE_FIREFOX;
```

Comment:
Added Firefox support for local testing due to issues with chromedriver (drivers for Chrome 115+ moved to new URL so need to update `wdio/*` to support new Chtome locally)